1-exercises/D-array-filter/exercise.js

```diff
@@ -8,7 +8,9 @@
 
 let pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-let pairsByIndex; // Complete this statement
+let pairsByIndex = pairsByIndexRaw.filter(
+  (elt) => Array.isArray(elt) && elt.length == 2
```

Comment:
This is nice Saadia, I like how you included the .length == 2 to specify a pair. I just used Array.isArray. Great work :)